recipe_scrapers/felixkitchen.py

```diff
@@ -0,0 +1,84 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._utils import normalize_string
+
+MARK_SEPARATOR = " "
+INGREDIENT_SEPARATOR = "• "
+
+
+class FelixKitchen(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "felix.kitchen"
+
+    def author(self):
+        return "FEL!X KITCHEN"
+
+    def title(self):
+        found = self.soup.find("meta", {"property": "og:title"})
+        return normalize_string(found.get("content"))
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return None
```

Comment:
We may want to emit a Python `warning` here, if we know that we're never going to find results for this field on this website.

Reply:
@jayaddison I've been following https://github.com/hhursev/recipe-scrapers/pull/1067, I'm not 100% sure what the verdict on the way you want this implemented is?

Reply:
I'm not certain either; whatever approach we choose, I want it to be simple (no more than a few lines), and minimally disruptive, because I think it's somewhat experimental but potentially useful information.

Reply:
@jayaddison I've now updated this with my attempt at an approach to `warnings` that allows for the warning to be silenced during testing.